src/helpers/dataFormatter.ts

```diff
@@ -10,6 +10,7 @@ const quizDataSchema = {
   quizId: 'QuizId',
   name: 'Name',
   isDraft: 'IsDraft',
+  userId: 'UserId',
```

Comment:
We are doing this because of Pascal case in db? Should we do something like we did for AL reader so that we don't have to hardcode all of these?

Reply:
fixed